Backend/Backend/Program.cs

```diff
@@ -88,36 +94,43 @@ public class Program
             };
         });
         builder.Services.AddAuthorization();
-        
+
         // Check if we are running in a docker container.
         bool inDockerEnv = Environment.GetEnvironmentVariable("DOTNET_RUNNING_IN_CONTAINER") == "true";
-       
+
         builder.Services.AddDbContext<AppDbContext>(options =>
         {
             // Set the connection string to the docker container if we are running in a docker container.
             if (inDockerEnv)
                 options.UseSqlServer(config.GetConnectionString("DockerConnection"));
-            else 
+            else
                 options.UseSqlServer(config.GetConnectionString("DefaultConnection"));
         });
 
 
         builder.Services.AddCors(o => o.AddPolicy("Dev-policy", builder =>
         {
             builder.WithOrigins("http://localhost:3000", "https://localhost:3000",
-            "http://localhost:3500", "https://localhost:3500","http://fronted:3500", "https://fronted:3500")
+            "http://localhost:3500", "https://localhost:3500", "http://fronted:3500", "https://fronted:3500",
```

Comment:
Small bit of hardcode here, but this is something we can fix later

Reply:
Thanks for exposing this...